Makefile

```diff
@@ -83,11 +93,15 @@ else
 # symbol table is required.
 
 ifeq ($(CONFIG_BUILD_LOADABLE),)
-
+ifeq ($(CONFIG_WINDOWS_NATIVE),y)
+$(BIN): $(foreach SDIR, $(CONFIGURED_APPS), $(SDIR)_all)
+	$(Q) for %%G in ($(CONFIGURED_APPS)) do ( $(MAKE) -C %%G archive )
+else
 $(BIN): $(foreach SDIR, $(CONFIGURED_APPS), $(SDIR)_all)
 	$(Q) for app in ${CONFIGURED_APPS}; do \
```

Comment:
what's problem is fixed by change app to %%G

Reply:
the usage of `FOR` loop in Windows is:
`FOR %variable IN (set) DO command [command-parameters]`
to use the FOR command in a batch program, specify `%%variable` instead of `%variable`

I tried to use `$(Q) for %%G in ($(CONFIGURED_APPS)) do ( $(MAKE) -C %%G archive )` or I tried to use `$(Q) for %%G in ($(CONFIGURED_APPS)) do ( $(MAKE) -C ${%%G} archive )` in Linux, but failed